README.md

```diff
@@ -1,2 +1,17 @@
 # new-project
```

Comment:
READMEs are for humans, you give an overview of what the repo does and how to use it